src/styles/index.scss

```diff
@@ -1,3 +1,133 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.card__image {
+  background-image: url("../images/imac.jpeg");
+  background-size: cover;
+  margin-top: 32px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  margin-top: 40px;
+  text-align: left;
+  color: #060b35;
+  font-weight: 500;
+  margin-bottom: 4px;
```

Comment:
it's a good practice to use one type of setting margin. 

So just move `margin-bottom: 4px;` from `.card__title` to `.card__product-code`  as `margin-top: 4px`